languages/grammar/pt.json

```diff
@@ -0,0 +1,48 @@
+{
+    "meta": {
+        "regExpFlags": "gi"
+    },
+    "v5": {
+        "em": [
+            ["^ A(s?) ", " na$1 "],
+            ["^ O(s?) ", " no$1 "],
+            
+            ["^ (Marginal|Parte|Passagem|Ponte|Radial|Rede|Servidão) ", " na $1 "],
```

Comment:
I'd recommend `Servid[ãa]o` for more robust matching

Reply:
I did see a couple instances of “Servidao” as a typo, though the total number of misspellings involving missing diacritics were far outnumbered by misspellings of “travessa” involving base letters, like “travesa” and “tavessa”. So I’m not sure if we should stop at diacritics or go further.

Reply:
Missing diacritic is common and expected mistake and could be just due to missing national keyboard. But fixing other typos on-fly is more risky IMHO